src/frontend/src/pages/Home/Home.view.tsx

```diff
@@ -162,6 +162,21 @@ export const HomeView = () => {
               </Button>
             </Link>
           </div>
+
+          <div>
+            <h3>JsLIGO</h3>
+            <p>JsLIGO has a syntax close to JavaScript, great for web developers.</p>
+            <img alt="js" src="/images/reason.svg" />
```

Comment:
I added a _reason.svg_ but would like to ask @AymericBethencourt to share sam initial templates, so I can generate the JS icon. Or maybe tell me what font was used there. I'll try to generate it on my own then.